pulp_container/app/registry.py

```diff
@@ -242,7 +242,6 @@ async def get_by_digest(self, request):
         distribution = self._match_distribution(path)
         self._permit(request, distribution)
         repository_version = distribution.get_repository_version()
-        log.info(digest)
```

Comment:
Good. We were logging a lot :)

Reply:
This really looks like forgotten debugging code (no accompanying explanation around the bare digest).